ros2node/ros2node/api/__init__.py

```diff
@@ -20,6 +20,19 @@
 HIDDEN_NODE_PREFIX = '_'
 
 NodeName = namedtuple('NodeName', ('name', 'namespace', 'full_name'))
+TopicInfo = namedtuple('Topic', ('name', 'types'))
+
+
+def parse_node_name(full_node_name):
+    tokens = full_node_name.split('/')
+    if 1 > len(tokens):
+        raise Exception('Invalid node name')
```

Comment:
I don't see any other uses of a bare `Exception` being thrown here.  This should probably be `RuntimeError` to conform to the rest of the code.  Also, it would be helpful to append the name of the invalid node to the error message.

Reply:
On it